components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/graaljs/JsGraalGraphBuilderFactory.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Update License

Reply:
Fixed with 8fdab59ae986de8a0620ae48bd7c9c8cffd4b391, here and similar places